utils/swift_build_support/tests/products/test_swift.py

```diff
@@ -91,6 +92,7 @@ def test_by_default_no_cmake_options(self):
             '-DSWIFT_FORCE_OPTIMIZED_TYPECHECKER:BOOL=FALSE',
             '-DSWIFT_STDLIB_ENABLE_STDLIBCORE_EXCLUSIVITY_CHECKING:BOOL=FALSE',
             '-DSWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING:BOOL=FALSE',
+            '-DSWIFT_ENABLE_EXPERIMENTAL_STRUCTURAL:BOOL=FALSE'
```

Comment:
There's a test failure:
```
17:26:32 Failed Tests (1):
17:26:32   Swift(linux-x86_64) :: Python/swift_build_support.swift
```
```
FAIL: Swift(linux-x86_64) :: Python/swift_build_support.swift (7300 of 13850)
******************** TEST 'Swift(linux-x86_64) :: Python/swift_build_support.swift' FAILED ********************
Script:
--
: 'RUN: at line 10';   /usr/bin/python3.5 /home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-main/swift/utils/swift_build_support/run_tests.py
--
Exit Code: 1

Command Output (stderr):
--
.......F......F...........................................................................................................
======================================================================
FAIL: test_by_default_no_cmake_options (tests.products.test_swift.SwiftTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-main/swift/utils/swift_build_support/tests/products/test_swift.py", line 98, in test_by_default_no_cmake_options
    self.assertEqual(set(swift.cmake_options), set(expected))
AssertionError: Items in the first set but not the second:
'-DSWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY:BOOL=FALSE'
'-DSWIFT_ENABLE_EXPERIMENTAL_STRUCTURAL:BOOL=FALSE'
Items in the second set but not the first:
'-DSWIFT_ENABLE_EXPERIMENTAL_STRUCTURAL:BOOL=FALSE-DSWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY:BOOL=FALSE'

======================================================================
FAIL: test_swift_runtime_tsan (tests.products.test_swift.SwiftTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-main/swift/utils/swift_build_support/tests/products/test_swift.py", line 116, in test_swift_runtime_tsan
    self.assertEqual(set(swift.cmake_options), set(flags_set))
AssertionError: Items in the first set but not the second:
'-DSWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY:BOOL=FALSE'
'-DSWIFT_ENABLE_EXPERIMENTAL_STRUCTURAL:BOOL=FALSE'
Items in the second set but not the first:
'-DSWIFT_ENABLE_EXPERIMENTAL_STRUCTURAL:BOOL=FALSE-DSWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY:BOOL=FALSE'

----------------------------------------------------------------------
Ran 122 tests in 0.074s

FAILED (failures=2)
```
It's because there's a missing comma on this line, and on line 113.
```suggestion
            '-DSWIFT_ENABLE_EXPERIMENTAL_STRUCTURAL:BOOL=FALSE,'
```

Reply:
Fixed!

Reply:
Thanks @dan-zheng, nice catch! 👍